docs/how-to/use-layers.md

```diff
@@ -0,0 +1,147 @@
+# How to use layers
+
+Managing multiple services across different environments becomes complex as systems scale. Pebble simplifies this with layered configurations, improving readability and maintainability.
+
+A base layer defines common settings (such as logging), while additional layers handle specific services or environment-specific overrides. This declarative approach, along with delegated layer management, allows for better cross-team collaboration and provides a clear view of each environment's configuration. For example, an operations team could manage base layers for logging, and service teams could manage layers for their services.
+
+## Pebble layers
+
+A layer is a configuration file that defines the desired state of services running on a system. 
+
+Layers are organized within a `layers/` subdirectory in the `$PEBBLE` directory. Their filenames are similar to `001-base-layer.yaml`, where the numerically prefixed filenames ensure a specific order of the layers, and the labels after the prefix uniquely identifies the layer. For example, `001-base-layer.yaml`, `002-override-layer.yaml`.
+
+A layer specifies service properties including the command to execute, startup behavior, dependencies on other services, and environment variables. For example:
+
+```yaml
+summary: Simple layer
```

Comment:
I think this can be significantly improved and shortened:

- We can probably drop the summary and description, as hardly anyone uses those. They are defined in the layer spec for people who really want to know.
- There are too many services and too fields. How about we have just two services, and drop `summary`, `startup`, and keep `requires` and `environment` (those are ones that are really used).
- And then maybe add a really simple health check, to show that it's not just services.
- In addition, could we make it a bit more real? Maybe have services `server` and `cron` or something.